packages/datadog-instrumentations/src/helpers/tx.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
```

Comment:
The original tx util from the old plugin system was not particularly easy to read when used. Now that it would mostly be used in instrumentation hooks, I'd say to remove the util completely and instead be more explicit with the code, so wrapping inline.